### ISSUE TITLE
If maxent is missing the package freezes

### ISSUE THREAD

**Issue by vijaybarve** [luismurao/nichetoolbox#7]
I guess the package needs to check first if `maxent` is available or not

**Comment by luismurao** [comment]
Right, Maybe after checking. If the package is not available show the directions (markdown or html format) of how to install it in dismo just like what I did to show how to play google motion charts when reviewing the html report

**Comment by luismurao** [comment]
[Check this out](https://github.com/luismurao/nichetoolbox/blob/master/inst/shinyApp/dismo_maxent/maxent_install.html)
Now the package checks if `maxent` is available for `dismo` and if not shows instructions to configure it.
